### DISCUSSION TITLE
How to pass options to rehype-autolink-headings

### DISCUSSION THREAD

**Discussion by vidyashv-carbon** [remarkjs/remark#1082] [Q&A] [answered]
I am using react-markdown 8.0.3 version in my project.I am trying to add anchor links to h2 header only. rehype-autolink-headings plugin will help how ever by default it does not add the link icon.
I want to use options.content and options.test propoerties for rehype-autolink-headings plugin to add link icon as in below screenshot.

<img width="237" alt="image" src="https://user-images.githubusercontent.com/64135464/204984735-57449314-72ef-4f42-9c6d-473f3ef27dad.png">

 **react-markdown accepts these plugins as array. How to pass the options object to rehype-autolink-headings plugin.?**
 
```

         <ReactMarkdown
            children={props.content}
            remarkPlugins={[remarkGfm]}
            rehypePlugins={[rehypeRaw,rehypeSlug,rehypeExternalLinks,rehypeAutolinkHeadings]}
          />
```

**Comment by wooorm** [comment, marked as answer]
https://github.com/remarkjs/react-markdown#use-a-plugin-with-options

In this case:

```js
<ReactMarkdown
  children={props.content}
  remarkPlugins={[remarkGfm]}
  rehypePlugins={[rehypeRaw,rehypeSlug,rehypeExternalLinks,[rehypeAutolinkHeadings, theOptions]]}
/>
```
